### ISSUE TITLE
Add licensing info in README.md

### ISSUE THREAD

**Issue by ghost** [mikepan/GameEngineBook#32]

**Comment by ghost** [comment]
@mikepan, maybe you will agreed use CC0 1.0 license for this book?
* https://creativecommons.org/publicdomain/zero/1.0/

**Comment by torvn77** [comment]
No, better use CC BY-SA https://creativecommons.org/licenses/by-sa/4.0/

Long live freedom!

**Comment by mikepan** [comment]
I think CC BY-SA works for me. One key issue is that we might want to publish the book on a different platform (possibly BlenderCloud).  I want to make sure everyone who contributed to this repo is okay with it.

**Comment by ghost** [comment]
@mikepan, please, add what version of the CC BY-SA license you mean and add link to text of this license in README.md :
* CC BY-SA 3.0 - http://creativecommons.org/licenses/by-sa/3.0/ 
* CC BY-SA 4.0 - http://creativecommons.org/licenses/by-sa/4.0/
